core-tests/src/test/java/org/mule/runtime/core/internal/el/DefaultExpressionManagerTestCase.java

```diff
@@ -254,12 +255,15 @@ public void parseLogMessage() throws MuleException {
     String expectedOutput =
         "current message is \norg.mule.runtime.core.internal.message.DefaultMessageBuilder$MessageImplementation\n{"
             + "\n  payload=test\n  mediaType=*/*\n  attributes=<not set>\n  attributesMediaType=*/*\n}";
-    assertThat(expressionManager.parseLogTemplate("current message is #[mel:message]", testEvent(), TEST_CONNECTOR_LOCATION,
-                                                  NULL_BINDING_CONTEXT),
-               is(expectedOutput));
-    assertThat(expressionManager.parseLogTemplate("current message is #[message]", testEvent(), TEST_CONNECTOR_LOCATION,
-                                                  NULL_BINDING_CONTEXT),
-               is(expectedOutput));
+    String melParsedTemplate = expressionManager
+        .parseLogTemplate("current message is #[mel:message]", testEvent(), TEST_CONNECTOR_LOCATION, NULL_BINDING_CONTEXT)
+        .replace("\r\n", "\n");
+    assertThat(melParsedTemplate, is(expectedOutput));
+
+    String dwParsedTemplate = expressionManager
+        .parseLogTemplate("current message is #[message]", testEvent(), TEST_CONNECTOR_LOCATION, NULL_BINDING_CONTEXT)
+        .replace("\r\n", "\n");
```

Comment:
Can we extract this to a function or, way better, a matcher?

Reply:
I think a matcher is a great idea. I'll go for it. It could also help us with the other "JSON" assertions on this PR, since those "JSON" are not strictly speaking compliant with the JSON format standard, therefore they'll demand more boilerplate code (besides the overhead) to compare them using tools like Google's Gson lib, for example.